tests/test_integration/test_commands/test_deploy.py

```diff
@@ -10,33 +10,6 @@
 from tests import data
 
 
-@pytest.mark.skipif(
-    sys.version_info < (3, 11), reason="We only run this test on Python 3.11+ to avoid parallelism issues"
-)
-def test_deploy_core_model(cdf_tool_config: CDFToolConfig) -> None:
-    """The motivation for this test is to ensure that we can deploy the core model
-    that has a lot of circular dependencies in the views. This was an issue up until
-    12. June 2024. The deployment failed as the server was not able to handle
-    reverse direct dependencies. This test is to ensure that the deployment works
```

Comment:
Core now exists and this issue has been solved. No need to run this expensive test.